pkg/log/logger.go

```diff
@@ -296,7 +296,7 @@ func NewLogger(opts ...Option) (*Logger, error) {
 			StacktraceKey:  "backtrace",
 			LineEnding:     zapcore.DefaultLineEnding,
 			EncodeLevel:    zapcore.LowercaseLevelEncoder,
-			EncodeTime:     zapcore.RFC3339NanoTimeEncoder,
+			EncodeTime:     o.timeEncoder,
```

Comment:
Will `EncodeTime` be a `nil` value if people don't call `WithTimeEncoder`?

Reply:
Yes, you're rignt. And it has  been fixed.